src/applications/registry.json

```diff
@@ -78,7 +78,7 @@
   },
   {
     "appName": "VA Profile",
-    "entryName": "profile-360",
+    "entryName": "profile",
```

Comment:
To platform reviewers, I believe this might be the only change that requires your approval 😄